<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttackForm</class>
 <widget class="QDialog" name="AttackForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>448</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>508</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Créer ou modifier une attaque</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="attackEffectsGroupBox">
     <property name="minimumSize">
      <size>
       <width>398</width>
       <height>394</height>
      </size>
     </property>
     <property name="title">
      <string>Effets de l'attaque</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_20">
      <item row="1" column="0">
       <widget class="QGroupBox" name="costBox">
        <property name="title">
         <string>Coût</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_71">
         <item>
          <widget class="QGroupBox" name="paCostBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_74">
            <item>
             <widget class="QLabel" name="paCostIcon">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons.qrc">:/assets/icons/skills/ap.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="paCostLabel">
              <property name="text">
               <string>PA</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="paCostSpin"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="pmCostBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_75">
            <item>
             <widget class="QLabel" name="pmCostIcon">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons.qrc">:/assets/icons/skills/mp.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pmCostLabel">
              <property name="text">
               <string>PM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pmCostSpin"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="pcCostBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_76">
            <item>
             <widget class="QLabel" name="pcCostIcon">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pcCostLabel">
              <property name="text">
               <string>PC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pcCostSpin"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="damagesHealBox">
        <property name="minimumSize">
         <size>
          <width>378</width>
          <height>105</height>
         </size>
        </property>
        <property name="title">
         <string>Dégâts et soins</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="3" column="0">
          <widget class="QGroupBox" name="damagesBox">
           <property name="minimumSize">
            <size>
             <width>176</width>
             <height>40</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="damagesIconMin">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons.qrc">:/assets/icons/skills/mastery.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="damagesSpinMin"/>
            </item>
            <item>
             <widget class="QLabel" name="damagesIconMax">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons.qrc">:/assets/icons/skills/mastery.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="damagesSpinMax"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QGroupBox" name="healBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="healIconMin">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons.qrc">:/assets/icons/skills/heals.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="healSpinMin"/>
            </item>
            <item>
             <widget class="QLabel" name="healIconMax">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons.qrc">:/assets/icons/skills/heals.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="healSpinMax"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="elementLabel">
           <property name="text">
            <string>Elément</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="elementComboBox">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="accessibleName">
            <string>0</string>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAlphabetically</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="attackNameBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_60">
         <item>
          <widget class="QLabel" name="attackNameLabel">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="attackNameSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="attackNameLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="launchBox">
        <property name="title">
         <string>Lancer</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="1" column="2">
          <widget class="QGroupBox" name="typeAreaBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_63">
            <item>
             <widget class="QLabel" name="typeAreaLabel">
              <property name="text">
               <string>Type de zone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="typeAreaFormComboBox">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="accessibleName">
               <string>0</string>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAlphabetically</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="typeAreaSizeSpin"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="typeLaunchBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_61">
            <item>
             <widget class="QLabel" name="typeLaunchLabel">
              <property name="text">
               <string>Type de lancer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="typeLaunchComboBox">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="accessibleName">
               <string>0</string>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAlphabetically</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="viewLineBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_62">
            <item>
             <widget class="QLabel" name="viewLineLabel">
              <property name="text">
               <string>Ligne de vue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="viewLineCheckBox">
              <property name="text">
               <string>Besoin</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="poAttackBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_68">
            <item>
             <widget class="QLabel" name="poAttackIcon">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../icons.qrc">:/assets/icons/skills/range.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="poAttackLabel">
              <property name="text">
               <string>PO</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="poAttackSpinMin"/>
            </item>
            <item>
             <widget class="QSpinBox" name="poAttackSpinMax"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="buttonBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Sauvegarder et quitter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
